ikasaneip/spec/service/housekeeping/src/main/java/org/ikasan/spec/harvest/HarvestingSchedulerService.java

```diff
@@ -0,0 +1,18 @@
+package org.ikasan.spec.harvest;
+
+/**
+ * Created by Ikasan Development Team on 24/08/2016.
+ */
+public interface HarvestingSchedulerService
+{
+
+    void registerJobs();
+
+    void removeJob(String jobName);
+
+    void addJob(String jobName);
+
+    void startScheduler();
```

Comment:
Is this expected to have its own scheduler instance or use the shared scheduler of the module ?

Reply:
Well the startJob and stopJob method are not in use in context of a module. Bear in mind that HarvestingSchedulerService is used by both Dashboard and module, but the two are using different implementations of HarvestingJob. 

Hence startJob and stopJob methods are utilised by dashboard not a module. 

In context of a module we are using shared scheduler.

Reply:
maybe just stick a comment in the method to the effect of "not required"